### PR TITLE
Punch card

### PR DESCRIPTION
**Pull request by saoudnazir**
Punch cards including Multi visit and Maintenance course has been added. Also, have been fixed like changing text colors, size of the images, using different relevant image for each course, 'cross-env' package added to run it on different platforms.

{ https://trello.com/c/14wNJGii }

### Done Checklist

* [x] Task has been completed as per the Trello card
* [ ] All acceptance criteria are met
* [x] Developer has tested this locally
* [x] Developer has reviewed their own PR
* [ ] Design has been reviewed by design team
* [ ] Tests have been added or updated
* [x] Views are responsive for mobile devices
* [ ] Feature files have been written or updated
* [ ] Changelog updated

### CODE REVIEW

File: meteor-app/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "private": false,
   "scripts": {
     "debug": "DEBUG=b2b:* MONGO_URL=mongodb://localhost:27017/b2b meteor --inspect --port 3030 --settings dev.settings.json",
-    "debug-local": "DEBUG=b2b:*  meteor --inspect --port 3030 --settings dev.settings.json",
+    "debug-local": "meteor --inspect --port 3030 --settings dev.settings.json",
```
Comment (mikkelking) [inline meteor-app/package.json]: The setting of the environment variable `DEBUG` is important here, and you should not just delete it. 
Please have a look at https://www.npmjs.com/package/cross-env and see if we can use it to support both Windows and Mac users

Comment (mikkelking) [reply meteor-app/package.json]: @saoudnazir Please also add this package to the project, and remove the number image files. Once done I can accept the PR and merge it (I will do these)

Comment (saoudnazir) [reply meteor-app/package.json]: @mikkelking Please have a look in Multi-Visits-Card. there is some error in there. It works on my machine but seems failed on circleci. I have removed number folder too.

Comment (mikkelking) [reply meteor-app/package.json]: You need to update the snapshots. You can do it with this command: 
`npm run test:snapshots -- -u` and the commit/push

You also need to resolve the conflict in `meteor-app/package-lock.json`, you can do this by doing a `git checkout -f package-lock.json`

**Pull request merged**